gm4_blast_furnaces/data/blast_furnaces/functions/main.mcfunction

```diff
@@ -0,0 +1,8 @@
+# Here is where all the main functions reside in
+
+
+execute as @e[tag=gm4_bf_output] at @s run function blast_furnaces:update

```

Comment:
All @e selectors should always have a `type` argument to help the selector resolution process eliminate as many entities as possible as quickly as possible.

Reply:
I'll look through the code. If setting the type works different then stuff like distance then it's probably better to have it. Else it would actually slow things down.